README.md

```diff
@@ -1 +1,12 @@
 # ELITE TEAM PERSONAL SKILLS
+
+## Mansoor Mustapha 0002
+
+### **General Skills**
+
+1. [Communication Skills](General_Skills_0002/Communication_Skills_%231_0002.md)
+
+### **Professional Skills**
+
+1. [Programming Languages](Professional_Skills_0002/Programming_Languages_%232_0002.md)
+2. [Web Development](Professional_Skills_0002/Web_Development_%232_0002.md)
```

Comment:
This Naming Convention is not convenient and does not match what expected

Reply:
Kindly, check my file in [Professional_Skills_0002](https://github.com/usanaphtal112/elite-git-practice-v2/tree/ch-Mansoor-0002/Professional_Skills_0002)
I used the recommended naming convention. I just made a slight change in Links not a file. You can check my first commit I used the recommended style, but it didn't function (leading to 404 error)
Kindly, check the commits here [My Commits](https://github.com/usanaphtal112/elite-git-practice-v2/compare/main...ch-Mansoor-0002)